css/style.css

```diff
@@ -502,3 +515,330 @@ form {
+  }
+
+  .home-content .supporting {
+    width: 920px;
+    height: 150px;
+    color: #172b4d;
+    font-size: 20px;
+    font-weight: 400;
+    margin-top: 2px;
+    margin-bottom: 0;
+    font-family: 'Inter', sans-serif;
+    padding: 0 60px;
+  }
+
+  .about-me-text h2 {
+    color: #172b4d;
+    font-weight: 700;
+    font-size: 48px;
+    font-style: normal;
+    font-family: 'Inter', sans-serif;
+    margin-bottom: 10px;
+    text-align: center;
+    width: 447px;
+    height: 48px;
+  }
+
+  .home-content .primary h2 {
+    text-align: center;
+    font-size: 40px;
+    line-height: 0.8em;
+    font-weight: 700;
+    color: #172b4d;
+    font-family: 'Inter', sans-serif;
+  }
+
+  .home-content .primary .software-dev {
+    font-size: 48px;
+  }
+
+  .social-icons {
+    margin-top: 0;
+    padding-top: 5px;
+  }
+
+  .my-recent-works {
+    margin-top: 500px;
+    width: 447px;
+    height: 80px;
+    padding-bottom: 0;
+    margin-bottom: 0;
+  }
+
+  .my-recent-works h3 {
+    color: #172b4d;
+    font-size: 30px;
+    font-weight: 700;
+    font-style: normal;
+  }
+
+  .my-recent-works img {
+    display: none;
+  }
+
+  .my-works {
+    min-width: 1171px;
+    min-height: 1020px;
+    margin: auto;
+  }
+
+  .works .work-1,
+  .works .work-2,
+  .works .work-3,
+  .works .work-4,
+  .works .work-5,
+  .works .work-6 {
+    max-width: 369px;
+    max-height: 474px;
+  }
+
+  .works {
+    display: grid;
+    gap: 12px;
+    grid-template-columns: 1fr 1fr 1fr;
+  }
+
+  .about-me {
+    width: 100%;
+    height: 345px;
+    margin: auto;
+    background: url("../images/Group\ desktop.svg");
+    background-repeat: no-repeat;
+    background-position: 1098px 0;
+  }
+
+  .about-me-text {
+    width: 100%;
+    height: 345px;
+    background: url("../images/Group\ desk.svg");
+    background-repeat: no-repeat;
+    background-position: 1px;
+  }
+
+  .about-me-text p {
+    font-weight: 400;
+    font-size: 20px;
+    color: #42526e;
+    margin-top: 10px;
+    width: 920px;
+    height: 96px;
+    padding-left: 350px;
+    padding-top: 20px;
+    text-align: center;
+  }
+
+  .about-me .get-resume {
+    margin-top: 60px;
+  }
+
+  .my-knowledge {
+    width: 1171px;
+    margin: auto;
+    display: grid;
+    gap: 2px;
+    grid-template-columns: repeat(3, 1fr);
+  }
+
+  .my-knowledge .languages,
+  .my-knowledge .frameworks,
+  .my-knowledge .skills {
+    width: 369px;
+    height: 375px;
+  }
+
+  .contact-form .form-flex {
+    margin: 0 80px;
+    display: grid;
+    width: 1171px;
+    height: 490px;
+    gap: 10px;
+    grid-template-columns: 1fr 2fr;
+  }
+
+  .contact-form .contact-me {
+    width: 463px;
+    height: 253px;
+  }
+
+  .contact-me-content {
+    height: 200px;
+    width: 463px;
+    padding: 0;
+    padding-right: 25px;
+    margin: 0;
+  }
+
+  .contact-me-content h3 {
+    color: #172b4d;
+    width: 463px;
+    font-weight: 700;
+    font-size: 30px;
+  }
+
+  form {
+    position: relative;
```

Comment:
- [ ] Kindly align the contact section in the centre as per the Figma design for the desktop version. Also, make sure to fix the background image.
Your design:
<img width="1720" alt="Screenshot 2022-03-24 at 11 09 58 PM" src="https://user-images.githubusercontent.com/70269507/160004798-64cbd6ce-0656-4714-971b-4b65d5337cc2.png">
Figma design: 
<img width="698" alt="Screenshot 2022-03-24 at 11 32 36 PM" src="https://user-images.githubusercontent.com/70269507/160005443-902ce971-8a90-4e60-bd7c-997041d4d633.png">